src/NovelAudioService.cpp

```diff
@@ -47,7 +45,7 @@ void NovelAudioService::load(std::string input, bool isMusic) {
     }
     else
     {
-      std::cerr << "ERROR: " <<Mix_GetError() << std::endl;
+      _console.log(std::string(Mix_GetError()), LogLevel::ERR);
```

Comment:
nit: Might be nice if there was an overload that automatically called `std::string` for the `errorCode`/`statusCode` case